### ISSUE TITLE
An execute's condition is checked for commands separately

### ISSUE THREAD

**Issue by MAttila42** [ExAtom/BetterDatapack#1]
If you do something like this:
```
/execute if entity @e[type=pig] run {
  /kill @e[type=pig]
  /say No more pigs...
}
```
Then the `/say` command won't run because the condition is checked for each command and it isn't true anymore.

Solution for the time being:
```
/execute if entity @e[type=pig] run /function namespace:main/kill_pigs

/mcfunction kill_pigs {
  /kill @e[type=pig]
  /say No more pigs...
}
```

After I fix this the conversion will do the same as shown above.

**Comment by MAttila42** [comment]
Since you don't give any name for executes, the created mcfunction file's names will be generated automatically. The default can be changed in `bdconfig.json`
New field's name: `executeNames` - default value: `exec`

**Comment by MAttila42** [comment]
Fixed in 1.4.0-pre-1. Won't close untill it's in a stable build too.

**Comment by MAttila42** [comment]
Fixed the executes and added `executeNames` config property in [1.4.0](https://github.com/ExAtom/BetterDatapack/releases/tag/1.4.0).
